newIDE/app/package.json

```diff
@@ -29,6 +29,7 @@
     "i18next": "^10.0.3",
     "keen-tracking": "1.1.3",
     "lodash": "4.17.4",
+    "loov-jsfx": "^1.2.0",
```

Comment:
Better enter just "1.2.0" to have the exact version specified. Otherwise, newer versions could break if something changed in the interface - and we would not notice it.